CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+Release v2.0.0 (2019-04-24)
+===========================
+### Changed
```

Comment:
I think we need to clearly state that there's a breaking change. Something like this is good example: https://github.com/angular/angular/blob/master/CHANGELOG.md#breaking-changes
And also it's better to add what package that's changed, for example: `event-logger: refactor log parameters`